### DISCUSSION TITLE
Problem with pc appearing on my console tacticalrmm

### DISCUSSION THREAD

**Discussion by Fabsaint** [amidaware/tacticalrmm#2094] [Q&A]
I see every day appearing new pc that we haven't added on our console ??
I send one pic attached to show ...
![probrmm_clientsghosts](https://github.com/user-attachments/assets/9910c988-4a97-4680-8e3d-37fba5e694f6)

**Comment by dinger1986** [comment]
https://docs.tacticalrmm.com/faq/#help-ive-been-hacked-and-there-are-weird-agents-appearing-in-my-tactical-rmm
